chainercv/datasets/image_folder_dataset.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+import os
+
+import chainer
+from chainercv.utils import read_image
+
+
+def find_label_names(root):
+    label_names = [d for d in os.listdir(root)
+                   if os.path.isdir(os.path.join(root, d))]
+    label_names.sort()
```

Comment:
Here is an example.
```
- root
    - 0
        - a.jpg
        - b.jpg
    - 1
        - c.jpg
    - 2
...
    - 10
        - y.jpg
        - z.jpg
```
In this case, user may want to sort directories numerical order (`0`, `1`, `2` ... `10`). Current code sort them by alphabetical order (`0`, `1`, `10`, `2`, ...).